src/AddIns/Uno.UI.MediaPlayer.Skia.X11/SharedMediaPlayerExtension.cs

```diff
@@ -195,21 +195,25 @@ public SharedMediaPlayerExtension(Windows.Media.Playback.MediaPlayer player)
 		VlcPlayer.Buffering += (o, a) => weakRef.GetTarget()?.OnBuffering(o, a);
 		VlcPlayer.Paused += (o, a) => weakRef.GetTarget()?.OnPaused(o, a);
 		VlcPlayer.TimeChanged += (o, a) => weakRef.GetTarget()?.OnTimeChanged(o, a); // PositionChanged fires way too frequently (probably every frame). We use TimeChanged instead.
+		VlcPlayer.VolumeChanged += (o, a) => weakRef.GetTarget()?.OnVlcVolumeChanged(o, a);
 
-		_vlcPlayerVolume = VlcPlayer.Volume;
+		if (PlatformHelper.IsWindows)
+		{
+			// Update the player volume from VLC volume, since all VLC instances have
+			// shared volume control on Windows.
+			OnVlcVolumeChanged(null, null!);
+		}
 
-		// For some reason, subscribing to VolumeChanged, even with an empty lambda, causes
-		// a native crash. Here's the crazy part: this only happens when a debugger is attached.
-		// This does not happen when a debugger is not attached even in debug builds. To work around
-		// this, we poll for the volume in OnTick instead.
-		// VlcPlayer.VolumeChanged += OnVolumeChanged;
```

Comment:
You have ignored this comment. When I tested on my side, your subscription crashes the app when a debugger is attached. We can stick with the original logic as it was, unless there's a problem with it.

Reply:
We will need to set this back. Even if it works on your end, it doesn't work on mine on two different machines. I've also seen independent non Uno-related reports of VLC hanging on VolumeChanged, so it's definitely not something specific to my setup.

Reply:
Sure, that could be useful. Thanks.